plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/server/N4JSRequestManager.java

```diff
@@ -13,16 +13,16 @@
 import java.util.concurrent.CancellationException;
 import java.util.concurrent.CompletableFuture;
 
-import org.eclipse.xtext.ide.server.concurrent.AbstractRequest;
-import org.eclipse.xtext.ide.server.concurrent.RequestManager;
+import org.eclipse.n4js.ide.xtext.server.concurrent.XAbstractRequest;
+import org.eclipse.n4js.ide.xtext.server.concurrent.XRequestManager;
 
 /**
  * Prints stack traces for requests that threw an exception
  */
-public class N4JSRequestManager extends RequestManager {
+public class N4JSRequestManager extends XRequestManager {
```

Comment:
Could you integrate this class into `XRequestManager`?

Reply:
👍